### PR TITLE
Allow KLOG_FORCE_FLUSH through sudo.

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR makes it easier to force log flushing for things like the updater which are run via sudo.

### CODE REVIEW

File: debian/postinst [hunk 1]
```diff
@@ -12,7 +12,8 @@ case "$1" in
     configure)
 
         # Create custom sudoers file
-        echo "%sudo   ALL=(root) NOPASSWD: /usr/bin/kano-launcher" > $TMP_FILE
+	echo "Defaults env_keep += KLOG_FORCE_FLUSH" > $TMP_FILE
```
Comment (tombettany) [inline debian/postinst]: There is something wrong with the indentation of this line - have we got a mix of tabs and spaces?

**Comment by Ealdwulf** [comment]
You're right, there was a tab. Now removed.

**Comment by alex5imon** [comment]
@skarbat please review

**Comment by skarbat** [comment]
looks good @Ealdwulf @alex5imon

**Comment by Ealdwulf** [comment]
Thanks; merging...

**Pull request merged**